config/default/kustomization.yaml

```diff
@@ -6,7 +6,9 @@ namespace: openshift-windows-machine-config-operator
 # "wordpress" becomes "alices-wordpress".
 # Note that it should also match with the prefix (text before '-') of the namespace
 # field above.
-namePrefix: windows-machine-config-operator-
+#
+# Setting this to an empty string in order to reduce resource name length
```

Comment:
There was a related topic where the operator pod and binary and called `manager` by default.
But the prefix was made to `""` purely because I got feedback that the resource names were made too long.

Reply:
Are any of the resource names changing with this transition?

Reply:
None should be